nautobot_golden_config/templates/nautobot_golden_config/generate_intended_config.html

```diff
@@ -8,6 +8,23 @@
     .button-container {
         margin-bottom: 24px;
     }
+    pre:has(code) {
```

Comment:
Why is this needed?

Reply:
Bootstrap has a default of 11.5px here so the `code` block inside of the `pre` block doesn't fill the box completely.

default

![image](https://github.com/user-attachments/assets/49c11a72-24aa-41c6-bdba-4128b4866c60)


with this fix

![image](https://github.com/user-attachments/assets/adefac09-a9ea-44d2-b67e-2582e3c484bf)